src/stravalib/protocol.py

```diff
@@ -238,7 +238,7 @@ def _token_expired(self) -> bool:
         """
         if self.token_expires:
             if time.time() > self.token_expires:
-                print("Your token has expired; Refreshing it now.")
+                logging.info("Your token has expired; Refreshing it now.")
```

Comment:
Oops - good catch @jsamoocha let's see if that change you suggested in the other PR quiets CI. 🤞🏻 If so, that fix will fix this pr as well! 🚀